manifests/mod/security.pp

```diff
@@ -140,6 +143,7 @@
   Optional[Array[String]] $custom_rules_set                    = $apache::params::modsec_custom_rules_set,
   Stdlib::Absolutepath $modsec_dir                             = $apache::params::modsec_dir,
   String $modsec_secruleengine                                 = $apache::params::modsec_secruleengine,
+  String $debug_log_level                                      = '0',
```

Comment:
Is this really a string and not an integer? https://www.feistyduck.com/library/modsecurity-handbook-free/online/ch04-logging.html suggests it should be
```suggestion
  Integer[0, 10] $debug_log_level                              = 0,
```